app/konnect/getting-started/index.md

```diff
@@ -28,7 +28,13 @@ title: Get started with Konnect
 
 ## About {{site.konnect_short_name}}
 
-{% include_cached /md/konnect/introducing-konnect.md %}
+{{site.konnect_short_name}} is an API lifecycle management platform that lets you build modern applications better, faster, and more securely. The control plane is hosted in the cloud by Kong, while the data plane is managed by you within your preferred network environment. All of this is powered by {{site.base_gateway}} — Kong's lightweight, fast, and flexible API gateway. 
+
+{{site.konnect_short_name}} can help you with the following use cases:
+
+* **Developers:** Use {{site.konnect_short_name}} as the easiest way to manage {{site.base_gateway}}. This helps you save time on boilerplate code to protect and secure APIs. 
+* **Small teams:** Use more advanced features, like API cataloguing and external developer portals, to monetize your APIs. 
+* **Large enterprise companies:** Use multi-geo support for the control plane and achieve federated API management with a single control plane for all teams in your company.
```

Comment:
@connorckong I left in the original paragraph since it's important context for how Konnect works at a big picture level. But I also added in a list below with the different use cases by persona. Let me know if these work!

I also left in the info about KIC and Mesh since it highlights the bigger picture and everything you can do with Konnect. I worry that if I left it out, users might think Kong Gateway is the only control plane you can use.

Reply:
@cloudjumpercat - think lines 36 and 37 need some tweaks. 36 may have a comma grammar issue and I think could benefit from a few more listed items such as detailed API analytics and saying, "and more"

Line 37 does not read quite right to me either. Control plane is repeated and it could probably stated as something like in the blog about giving central teams governance tools while giving API teams speed and flexibility "https://konghq.com/blog/enterprise/federated-api-management"